src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -78,4 +79,14 @@ private void outputValidate(EiffelEventType eiffelType, String jsonStringInput)
         JsonObject jsonObject = new JsonParser().parse(jsonStringInput).getAsJsonObject();
         validator.validate(jsonObject);
     }
+
+    @Override
+    public String getEventId(JsonObject json) {
+        if (json.isJsonObject() && json.getAsJsonObject().has(RemremSemanticsConstants.META) && json.getAsJsonObject()
+                .getAsJsonObject(RemremSemanticsConstants.META).has(RemremSemanticsConstants.ID)) {
```

Comment:
I would propose a static import of META and ID to remove the need of writing RemremSemanticsConstants. several times. Will improve readability.